tech-far-hub/src/components/page-layout-nav.tsx

```diff
@@ -78,13 +81,15 @@ const PageLayoutNav: React.FC<IPageLayoutNav> = ({
           atCurrent = true;
           return (
             <>
-              <a href="#" className="usa-current" key="current">
-                {node.frontmatter.heading}
-              </a>
+              {showSiblings === true && (
```

Comment:
if showSIblings is boolean (which it seems it is) this might be able to be simplified from this:

`{showSiblings === true  && (`

to this:
`{showSiblings && (`

Reply:
I've been running into a funny problem where sometimes gatsby will like show the value unless I set an explicit comparison. Lemme try tho!